selenium_tests/test_ui_unit_field.py

```diff
@@ -0,0 +1,36 @@
+# To run these tests, it is required that you perform the following two actions first:
+#   1. Install selenium webdriver and add it to your path variable
+#       https://zwbetz.com/download-chromedriver-binary-and-add-to-your-path-for-automated-functional-testing/
+#   2. Start the server running locally at the URL "http://localhost:8000/"
```

Comment:
This information is now in the README, so it isn't necessary for it to be in every file any longer.